src/heaps.jl

```diff
@@ -37,8 +37,8 @@
 #   - update!(h, i, v)    updates the value of an element
 #                         (referred to by the handle i)
 #
-#   - delete!(h, id)      deletes the node present at
-#                         node_index id from the heap
+#   - delete!(h, i)      deletes the node present at
+#                         index i from the heap
```

Comment:
Since the other functions in this group (`push!`, `update!`, and `top_with_handle`) refer to `i` as a `handle`, let's do it here as well.